superset/assets/src/explore/components/controls/DatasourceControl.jsx

```diff
@@ -116,23 +118,16 @@ class DatasourceControl extends React.PureComponent {
 
   render() {
     const { menuExpanded, showChangeDatasourceModal, showEditDatasourceModal } = this.state;
-    const { datasource, onChange, onDatasourceSave, value } = this.props;
+    const { datasource, onChange, onDatasourceSave, value, canChangePhysicalTable } = this.props;
     return (
       <div>
         <ControlHeader {...this.props} />
         <div className="btn-group label-dropdown">
-          <OverlayTrigger
-            placement="right"
-            overlay={
-              <Tooltip id={'error-tooltip'}>{t('Click to edit the datasource')}</Tooltip>
```

Comment:
I think we should keep this, but on click it should bring up the `Change Datasource` dialog instead since that's a pretty common action. The tooltip would then be changed to say `Click to change the datasource` or `Click to select a datasource` or something like that

Reply:
fixed. Keep overlay trigger, and tooltip is `Click to change the datasource`.